packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/DestinationNetworkBox.tsx

```diff
@@ -97,7 +101,9 @@ function BalanceRow({
         <span>Balance: </span>
         <span
           aria-label={`${symbol} balance amount on ${
-            isDepositMode ? 'childChain' : 'parentChain'
+            transferMode === 'deposit' || transferMode === 'teleport'
+              ? 'childChain'
+              : 'parentChain'
```

Comment:
Not changed in this PR, but it would be nice to name it in a more user friendly way?

Reply:
do you mean `childChain` and `parentChain`? 

this is used in e2e tests so i'm not changing it but good point that we can discuss with the team to update it